content/handbook/support/workflows/dedicated.md

```
@@ -0,0 +1,210 @@
+#### Switchboard でのテナントステータスの理解
+
+Switchboard の Overview ページには、各 GitLab Dedicated インスタンスのリアルタイムステータス情報が表示されます。Dedicated Switchboard ワークフローの [テナントステータスインジケータの理解](dedicated_switchboard.md#understanding-tenant-status-indicators) を参照してください。
+
+#### Dedicated インシデントを発生させる
+
+GitLab Dedicated インシデントを発生させる際には、3 つすべてのステップを完了する必要があります:
+
+1. [Slack 経由で Dedicated インシデントを宣言](../../engineering/infrastructure-platforms/incident-management/_index.md#reporting-an-incident) します。
+   - [Dedicated Switchboard](dedicated_switchboard.md#accessing-switchboard) を参照して、顧客の `internal_reference` と `tenant_id` (識別子としても知られる) を提供します。
+1. Slack で自動的に作成されたインシデントチャンネルで、現在の状態のサマリーを提供します。
+1. 宣言された Dedicated インシデントは、GitLab.com とは異なり、自動的に EOC をページしません。[インシデントを GitLab Dedicated エンジニアオンコールにエスカレーションする](#escalate-to-gitlab-dedicated-engineer-on-call) に進んでください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)